backend/scripts/debug_ai_processing.py

```
@@ -0,0 +1,316 @@
+
+class AIProcessingDebugger:
+    """AI处理调试器"""
+
+    def __init__(self):
+        self.chat_service = ChatService()
+        self.pipeline = PreprocessingPipeline(self.chat_service)
+
+    async def debug_content_processing(self, content_id: str = None):
+        """调试内容处理流程"""
+        print("🔍 开始AI处理调试...")
+        print(f"📊 环境信息:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)